parts/windows/kuberneteswindowssetup.ps1

```diff
@@ -188,6 +188,11 @@ $global:TLSBootstrapToken = "{{GetTLSBootstrapTokenForKubeConfig}}"
 # Base64 representation of ZIP archive
 $zippedFiles = "{{ GetKubernetesWindowsAgentFunctions }}"
 
+$useContainerD = ($global:ContainerRuntime -eq "containerd")
+$global:KubeClusterConfigPath = "c:\k\kubeclusterconfig.json"

```

Comment:
just curious can we add tests for the ps1 file?

Reply:
We should be able to do it but it needs much work. At first we need to generate values and compile this script since it actually is a go template.

Reply:
I tested these change in AKS RP side